swarm/network/simulation/node.go

```diff
@@ -117,88 +89,10 @@ func (s *Simulation) AddNodes(count int, opts ...AddNodeOption) (ids []enode.ID,
 	return ids, nil
 }
 
-// AddNodesAndConnectFull is a helpper method that combines
-// AddNodes and ConnectNodesFull. Only new nodes will be connected.
-func (s *Simulation) AddNodesAndConnectFull(count int, opts ...AddNodeOption) (ids []enode.ID, err error) {
-	if count < 2 {
-		return nil, errors.New("count of nodes must be at least 2")
-	}
-	ids, err = s.AddNodes(count, opts...)
-	if err != nil {
-		return nil, err
-	}
-	err = s.Net.ConnectNodesFull(ids)
-	if err != nil {
-		return nil, err
-	}
-	return ids, nil
-}
-
-// AddNodesAndConnectChain is a helpper method that combines
-// AddNodes and ConnectNodesChain. The chain will be continued from the last
-// added node, if there is one in simulation using ConnectToLastNode method.
-func (s *Simulation) AddNodesAndConnectChain(count int, opts ...AddNodeOption) (ids []enode.ID, err error) {
```

Comment:
the `AddNodeOption` should stay. we need to be able to control the configurations of the nodes that are being initialised

Reply:
I've found no use case for that in the actual code. Do we have something in progress where we need that `opts` argument?

Reply:
https://github.com/ethersphere/go-ethereum/pull/1077#discussion_r243821759

Reply:
Oh, I've read that but I did not connect the two. Still, there we say "maybe in the future" and I would say [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it) on that . If that is not too harsh from me. :)

Reply:
The commented code was removed eventually. However, `AddNodeOption` was brought back for `AddNode()` and `AddNodes()`.